site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,112 @@
+
+- FusionAuth Admin
+----
+
+Save your template by clicking the blue save icon.
+
+When a user clicks the `Forgot Password` link on the login page or calls the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[forgot password API], this template will be used to build the email sent to the user. This is the start of the workflow to change their password. This template is used so long as the application doesn't have a `Forgot Password` template configured.
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview]. After, log in to your application by clicking on `Applications` in the navigation sidebar, selecting the green magnifying glass icon, and copying and pasting the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+=== Configuring Application Email Templates
+
+Finally, you'll override your generic Tenant Email Template you just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the blue edit icon for the `Forgot Password` email template and copy the `HTML Template` `Text Template` sections into a notepad.
+
+Create a new email template by going to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste your default template from above into the `HTML Template`  and `Text Template` sections. Feel free to edit these templates based on your application.
+
+Save your template by clicking the blue save icon in the top right corner.
+
+Now configure the email template by navigating to [breadcrumb]#Applications -> Your Application -> click the green edit icon -> Email -> Templates -> Forgot password# and select your application email template - this option will have the same name you populated the `Name` field with from above. Click the blue save icon in the top right corner.
+
+=== Test the Custom Template
+
+For our demo application Pied Piper, we'll call the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] for a user. You can also use the Login URL, as previously demonstrated.
+```shell
+curl --request POST \
+    https://YOUR_FUSIONAUTH_INSTANCE/api/user/forgot-password \
+  --header 'Authorization: YOUR_API_KEY' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "applicationId": "85a03867-dccf-4882-adde-1a79aeec50df",
+    "loginId": "dinesh@piedpiper.com",
+     "sendForgotPasswordEmail": true
+}'
+```
+
+Modify the curl command for your `applicationId`, `YOUR_API_KEY`, `YOUR_FUSIONAUTH_INSTANCE`, and `loginId` with the email address of the user you wish to send the email to. Remember this user must be registered for the application before the email can be sent.
```

Comment:
```suggestion
Modify the curl command. You'll need to update a number of values.

* Replace `YOUR_API_KEY` with a valid FusionAuth API key.
* Replace `YOUR_FUSIONAUTH_INSTANCE` with the URL to your instance. If running locally this will be `http://localhost:9011`.
* Replace the value of `applicationId` with the Id of your application.
* Replace the value of `loginId` with the email address of the account for which the password is being reset. Remember this user must be registered for the application before the email can be sent.
```